src/OpenLoco/Paint/Paint.h

```diff
@@ -119,6 +119,10 @@ namespace OpenLoco::Paint
         [[nodiscard]] Ui::ViewportInteraction::InteractionArg getStationNameInteractionInfo(const uint32_t flags);
         [[nodiscard]] Ui::ViewportInteraction::InteractionArg getTownNameInteractionInfo(const uint32_t flags);
         Gfx::drawpixelinfo_t* getContext() { return _dpi; }
+        uint8_t rotation() { return currentRotation; }
+        void setCurrentObject(void* object) { _currentObject = object; }
```

Comment:
`Object` is a bit confusing, considering we use the same term for .DAT files. How about 'Item'? or 'Subject'?

Reply:
Subject would probably make most sense